site/gatsby-site/src/global.css

```diff
@@ -911,3 +911,16 @@ img {
     padding-left: 0px;
   }
 }
+
+/* disable Github Permalink */
+.bytemd-toolbar-right [bytemd-tippy-path='5'] {
```

Comment:
What is this doing?

Reply:
The Markdown Editor has a button that links to the GitHub project, and that's the only way to hide it at the moment.